staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -74,6 +70,21 @@ func (p *policyChecker) LevelAndStages(attrs authorizer.Attributes) (audit.Level
 	return DefaultAuditLevel, p.OmitStages
 }
 
+func (p *policyRuleEvaluator) IsOmitManagedFieldsEnabled() bool {
+	// If we can't decide, return false to maintain current behavior which is
+	// to retain the manage fields in the audit.
+	value, exists := p.Policy.Annotations["audit.kubernetes.io/omit-managed-fields"]
+	if !exists {
+		return false
+	}
+
+	if omitManagedFields, err := strconv.ParseBool(value); err == nil {
```

Comment:
is this re-parsing the policy annotation on every check of every request? can we do this once since the policy isn't dynamic?

Reply:
@liggitt thanks for the suggestion, I have moved the parsing logic to the construction of of the policy rule  evaluator interface.